File: codes/ABC/ABC166/ABC166_B.rb
n, k = gets.split.map(&:to_i)
snk = Array.new(n+1, 0)
for i in 0..(k-1)
  d = gets.to_i
  a = gets.split.map(&:to_i)
  a.each{|an| snk[an] += 1}
end
cnt = 0
for i in 1..n
  cnt += 1 if snk[i] == 0
end
puts cnt

File: codes/ABC/ABC166/ABC166_B.py
n, k = map(int, input().split())
snk = [1]*n
for i in range(k):
  d = int(input())
  for a in (map(int, input().split())):
    snk[a-1] = 0
cnt = 0
for i in snk:
  cnt += i
print(cnt)